etlman/projects/tests/test_views.py

```diff
@@ -37,14 +57,17 @@ def test_update_step_in_db(self):
         - POST for new object with valid data returns redirect, saved to DB
         - POST for existing object with valid data returns redirect, updates saved to DB
         """
+        client = get_authenticated_client()
         step_model = StepFactory.build(pipeline=PipelineFactory())
-        self.__get_response_from_post_data_to_view(step_model)
+        response = self.get_response_from_post_data_to_view(client, step_model)
+        html = str(response.content)
 
+        assert Step.objects.count() == 1, html
```

Comment:
I am not sure the HTML will help debug if this fails; maybe show all the step objects instead?
```suggestion
        assert Step.objects.count() == 1, Step.objects.all()
```